microprofile/WFLY-13048_upgrade_microprofile-health_2.2.adoc

```diff
@@ -0,0 +1,59 @@
+= Upgrade MicroProfile Health to 2.2
+:author:            Jeff Mesnil
+:email:             jmesnil@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          microprofile,health,observability
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+
+MicroProfile 3.3 provides a minor upgrade for https://github.com/eclipse/microprofile-health/releases/tag/2.2[MicroProfile Health 2.2].
+
+This component upgrade also covers the upgrade of smallrye-health that implements MicroProfile Health 2.2.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-13048[WFLY-13048]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1426[EAP7-1426]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* TBD
+
+=== Testing By
+
+[X] QE
+
+== Requirements
+
+* Upgrade MicroProfile Health to 2.2
+* Upgrade smallrye-health to the version that implements MicroProfile Health 2.2
```

Comment:
Those two look more like a detail/bullet for "Implementation plan" section, IMO. WDYT?

Reply:
you're right, I'll update the doc